disputer/test/Disputer.js

```diff
@@ -97,11 +98,12 @@ contract("Disputer.js", function(accounts) {
     await syntheticToken.approve(emp.address, toWei("100000000"), { from: liquidator });
     await syntheticToken.approve(emp.address, toWei("100000000"), { from: disputeBot });
 
-    // Create a new instance of the ExpiringMultiPartyClient to construct the disputer
+    // Create a new instance of the ExpiringMultiPartyClient & GasEstimator to construct the disputer
     empClient = new ExpiringMultiPartyClient(ExpiringMultiParty.abi, web3, emp.address);
+    gasEstimator = new GasEstimator();
```

Comment:
where do you first declare `gasEstimator`? Maybe add below line 39: `let gasEstimator`

Reply:
it is not declared anywhere, just like `empClient` and `disputer` are not declared anywhere.

Reply:
oh ok, as long as you're being consistent